scripts/trace_flowcell.py

```diff
@@ -2,117 +2,145 @@
 import requests
 import json
 import sys
+import subprocess
 import yaml
 import argparse
 from datetime import datetime
 
+class get_stuff:
+    def __init__(self, api_base_url, access_headers, req_verify, tag):
+        self.api_base_url = api_base_url
+        self.headers = access_headers
+        self.verify = req_verify
+        self.tag = tag
+    
+    def get_traces_for_tag(self):
+        traces_response = requests.get(
+            "{}/{}/?{}={}".format(self.api_base_url, "traces", "trace_tag", self.tag),
+            verify = self.verify,
+            headers = self.headers)
+        if not traces_response.ok:
+            raise Exception("Response not OK, got: " + traces_response.text)
+        self.traces = json.loads(traces_response.text)
+        return json.loads(traces_response.text)
```

Comment:
this is of course a design choice but I would probably not store the traces as an attribute but instead just return them and pass them as attributes to the next method. the reason is that this way you will be dependent on the state of the class instance, i.e. that you have called the methods in the correct order

Reply:
I think it's great to return a generator in the same way as below, i.e. something like:

```suggestion
        for trace in json.loads(traces_response.text):
            yield trace
```